test/Microsoft.AspNetCore.Mvc.Core.Test/Builder/MvcEndpointInfoBuilderExtensionsTest.cs

```diff
@@ -31,13 +27,6 @@ public void MapAreaEndpoint_Simple()
 
             Assert.Null(endpointInfo.Name);
             Assert.Equal("site/Admin/", endpointInfo.Template);
-            Assert.Collection(
-                endpointInfo.Constraints.OrderBy(kvp => kvp.Key),
```

Comment:
Currently MatchProcessorReferences do not expose any data for us to do some validation